ci/jenkins/test.sh

```diff
@@ -292,6 +293,43 @@ function deliver_antrea_windows {
 
     git show --numstat
     make clean
+}
+
+function revert_snapshot_windows {
+    echo "==== Reverting Windows VM ${WIN_NAME} ====="
+    govc snapshot.revert -vm ${WIN_NAME} win-initial
+    # If Windows VM fails to power on correctly in time, retry several times.
+    winVMIPs=""
+    for i in `seq 10`; do
+        winVMIPs=$(govc vm.ip -wait=2m -a ${WIN_NAME})
+        if [[ $winVMIPs != "" ]]; then
+            echo "Windows VM ${WIN_NAME} powered on"
+            break
+        fi
+        echo "Windows VM ${WIN_NAME} failed to power on"
+        govc vm.power -on ${WIN_NAME} || true
+    done
+    if [[ $winVMIPs == "" ]]; then
+        echo "Windows VM ${WIN_NAME} didn't power on after 10 tries, exiting"
+        exit 1
+    fi
+    IP=$(kubectl get node "${WIN_NAME}" -o jsonpath='{.status.addresses[0].address}')
```

Comment:
could this get IP from winVMIPs to be more generic? (so it can be used for jumper too)

Reply:
Yes, updated it to be a generic function